persistent/timestamp.py

```diff
@@ -79,8 +79,10 @@ def _parseRaw(octets):
     second = b * _TS_SECOND_BYTES_BIAS
     return (year, month, day, hour, minute, second)
 
+TimeStampPy = None
```

Comment:
This could use a comment explaining that it'll be overwritten by the `@use_c_impl` decorator and is here solely to prevent linter warnings, or whatever it's actually here for.